interpreter/CMakeLists.txt

```diff
@@ -249,7 +260,8 @@ else()
       "--prefix"
       "--src-root"
       "--cmakedir"
-      "--build-mode")
+      "--build-mode"
+      "--cppflags")
```

Comment:
Doesn't that show more than just the `-D` flags?

Reply:
Yes, you are right, it returns something like
```
-I/path/to/llvm/include -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS
```
For sure, we could filter the list:
```cmake
string(REPLACE " " ";" LLVM_DEFS "${LLVM_DEFS}")
list(FILTER LLVM_DEFS INCLUDE REGEX "^-D")
string(REPLACE ";" " " LLVM_DEFS "${LLVM_DEFS}")
```